### PR TITLE
[IMP] web_editor: add animated shapes

### PR DESCRIPTION
**Pull request by Brieuc-brd**
The purpose is to update the shapes of themes depending on the PR [#70843.](https://github.com/odoo/odoo/pull/70843)
Also, it set correctly the colors of the shapes that use the colors of the header or the footer.

Task-2319488

### CODE REVIEW

File: theme_artists/views/snippets/s_call_to_action.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
         </xpath>
         <!-- Shape -->
         <xpath expr="//div[hasclass('container')]" position="before">
-            <div class="o_we_shape o_web_editor_Airy_12 o_we_flip_y"/>
+            <div class="o_we_shape o_web_editor_Airy_12 o_we_flip_y o_footer_extra_shape_mapping"/>
```
Comment (qsm-odoo) [inline theme_artists/views/snippets/s_call_to_action.xml]: This should not be done, the point is that this class is automatically added when building pages **through the new website configurator**. When using the editor, you don't want that class when dropping a snippet into the page, otherwise the shape would be linked to the footer color wherever it is dropped.

File: theme_paptic/static/src/scss/primary_variables.scss [hunk 2]
```diff
@@ -84,6 +84,6 @@ $o-color-palettes-compatibility-indexes: (
 // Shapes
 //------------------------------------------------------------------------------
 
-$o-bg-shapes: change-shape-colors-mapping('web_editor', 'Blocks/04', (3: 1));
+$o-bg-shapes: change-shape-colors-mapping('web_editor', 'Blocks/04', (3: 1, 1: 2));
```
Comment (qsm-odoo) [inline theme_paptic/static/src/scss/primary_variables.scss:87]: Should this be done in this commit? I don't see Blocks/04 changed in the community PR, is this a fix? If yes, it should be done in another PR.

File: theme_paptic/views/customizations.xml [hunk 3]
```diff
@@ -206,11 +206,11 @@
 <!-- ==== Three Columns ===== -->
 <template id="s_three_columns" inherit_id="website.s_three_columns" name="Paptic s_three_columns">
     <xpath expr="//section" position="attributes">
-        <attribute name="class" remove="bg-200 pt32 pb32 o_cc o_cc3" add="pt120 pb120" separator=" "/>
-        <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/01","flip":[]}</attribute>
+        <attribute name="class" remove="bg-200 pt32 pb32" add="pt120 pb120 o_cc1 bg-o-color-2" separator=" "/>
```
Comment (qsm-odoo) [inline theme_paptic/views/customizations.xml:209]: This will be in conflict with another task we have already fixing/chaning that. But for the record, what you do here is not ok: you cannot have a o_ccX class and a bg-o-color-X class at the same time, so remove this change :wink:

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

File: theme_artists/views/snippets/s_call_to_action.xml (hunk 1 above)
Comment (Brieuc-brd) [reply theme_artists/views/snippets/s_call_to_action.xml]: Ok I thought it was an issue, but yeah it's logical in fact :). I removed the [FIX] commit.

File: theme_paptic/static/src/scss/primary_variables.scss (hunk 2 above)
Comment (Brieuc-brd) [reply theme_paptic/static/src/scss/primary_variables.scss]: The render of this snippet is not the same than the screenshot (paptic_screenshot.jpg).
We'll probably update that in another PR with some other updates on this theme :). I removed the [FIX] commit.

File: theme_paptic/views/customizations.xml (hunk 3 above)
Comment (Brieuc-brd) [reply theme_paptic/views/customizations.xml]: @qsm-odoo 
Ok I removed the change. But without that, the shape is hidden (it's white on a white background).
Should I let like that ?

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+
